common/app/common/configuration.scala

```diff
@@ -136,17 +144,7 @@ class GuardianConfiguration extends Logging {
       configuration.getIntegerProperty("tag_indexes.rebuild_rate_in_minutes").getOrElse(60)
   }
 
-  object environment {
-    private val installVars = new File("/etc/gu/install_vars") match {
-      case f if f.exists => IOUtils.toString(new FileInputStream(f))
-      case _ => ""
-    }
-
-    private val properties = Properties(installVars)
-
-    def apply(key: String, default: String) = properties.getOrElse(key, default).toLowerCase
-
-    val stage = apply("STAGE", "unknown")
+  object environment extends InstallVars.InstallationVars {
```

Comment:
so basically to be more concrete instead of the general comments I made, I'd suggest basically keep everything as you've done it is great, and just take this `extends InstallVars.InstallationVars` away and either make a lazy val stage to point to the InstallVars object, or make changes to all the 23 places if you prefer.  Then you've avoided having to extend other traits.